### PR TITLE
feat(translation): add Thai translations 

### PR DESCRIPTION
**Pull request by ooHAoo**
| Question         | Response    |
| ---------------- | ----------- |
| Version?         | latest      |
| Devices tested?  | Android |
| Bug fix?         | no      |
| New feature?     | yes      |
| Includes tests?  | no      |
| All Tests pass?  | yes      |
| Related ticket?  | #439        |

---

## Description

Added Thai translation.

Sorry it took a while to get this pull request. 
Had to borrow a newer phone to test since I have limited storage with my own device.
Please let me comments on anything i should change up!

### CODE REVIEW

File: .all-contributorsrc [hunk 1]
```diff
@@ -827,6 +827,13 @@
       ]
     },
     {
+      "login": "ooHAoo",
+      "name": "Huda Al Dallal",
+      "avatar_url": "https://avatars2.githubusercontent.com/u/26858034?s=400&u=fd8893b7f81dd9c9baeaf2807a1f7f98db0b2fe2&v=4",
+      "profile": "https://github.com/ooHAoo",
+      "contributions": [
+        "translation",
+        "tool"
```
Comment (chinesedfan) [inline .all-contributorsrc]: I saw `tool` rarely. Can you explain a little more about it?

Comment (ooHAoo) [reply .all-contributorsrc]: This was added accidentally and I thought I did fix it.
Sorry about that, will remove that once again.

File: package.json [hunk 2]
```diff
@@ -112,7 +112,7 @@
     "babel-preset-flow": "^6.23.0",
     "babel-preset-react-native": "1.9.1",
     "babylon": "^6.18.0",
-    "chalk": "^2.3.2",
+    "chalk": "^2.4.1",
```
[the remark this answers is not in the document]
Comment (ooHAoo) [reply package.json]: I believe this was changed when I was doing some testings.

File: package.json [hunk 3]
```diff
@@ -129,7 +129,7 @@
     "eslint-plugin-react": "^7.1.0",
     "eslint-plugin-react-native": "^2.3.2",
     "flow-bin": "0.65.0",
-    "glob": "^7.1.2",
+    "glob": "^7.1.3",
```
Comment (chinesedfan) [inline package.json]: Sorry, I missed a similar place.

**Comment by lex111** [comment]
@ooHAoo thank you for the translation! Happy Hacktoberfest! :jack_o_lantern:

**Pull request merged**